### PR TITLE
Feature/add default services to form

### PR DESCRIPTION
**Pull request by annashamray**

### CODE REVIEW

File: src/bptl/tasks/tests/test_add_defaultservices.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    def setUp(self):
+        self.client.login(username="super", password="letmein")
+
+        patcher = patch("bptl.tasks.forms.register", new=test_register)
+        self.mocked_register = patcher.start()
+        self.addCleanup(patcher.stop)
+
+        self.url = reverse_lazy("tasks:taskmapping-create")
+
+        self.data = {
```
Comment (sergei-maertens) [inline src/bptl/tasks/tests/test_add_defaultservices.py]: For the future: django-webtest makes your life so much easier when dealing with forms/formsets :)

File: src/bptl/tasks/views.py [hunk 2]
```diff
@@ -19,3 +19,29 @@ class AddTaskMappingView(LoginRequiredMixin, CreateView):
     model = TaskMapping
     form_class = TaskMappingForm
     success_url = reverse_lazy("tasks:taskmapping-list")
+
+    def get_formset(self):
```
Comment (sergei-maertens) [inline src/bptl/tasks/views.py]: django-extra-views has some useful base views to deal with formsets, please look at that next time :)

File: src/bptl/tasks/tests/test_add_defaultservices.py (hunk 1 above)
Comment (annashamray) [reply src/bptl/tasks/tests/test_add_defaultservices.py]: I know, but when I tried to use `django-webtest` in response.form the `service` fields were missing (`alias` fields were there though). In browser everything was ok, so I just switched to django testcases

File: src/bptl/tasks/views.py (hunk 2 above)
Comment (annashamray) [reply src/bptl/tasks/views.py]: Thanks! I'll tasks a look

**Pull request merged**